src/twisted/python/test/test_release.py

```diff
@@ -47,42 +50,9 @@
 )
 
 if os.name != "posix":
```

Comment:
you should be able to get mypy to infer the type with:
```python
skip = (
    None
    if os.name == "posix"
    else "Release toolchain only supported on POSIX."
)
```

also I think `sys.platform != "win32"` is preferred?

https://gitter.im/python-trio/general?at=5da80653c87a1d28acc46108

Reply:
Thanks. I will apply the changes